library/src/main/java/com/devbrackets/android/playlistcore/service/PlaylistServiceCore.java

```diff
@@ -738,6 +738,20 @@ protected void performPlay() {
                 videoPlayer.play();
             }
         }
+        
+        //Immediately pauses the media
```

Comment:
This shouldn't be moved here, the correct place would be in the `startMediaPlayer()` function.  Or more appropriately, clean up the methods called from `startMediaPlayer()` to correctly abide by the seek and immediately pause (which should be renamed to `startPaused` with the re-work).

Moving these methods here will cause the checks to get called every time the user clicks play (e.g. after pausing)